packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -22,6 +22,16 @@ export interface GetMiddlewareOptions {
   bodyParserConfig?: OptionsJson | boolean;
   onHealthCheck?: (req: express.Request) => Promise<any>;
   disableHealthCheck?: boolean;
+  // There's no real point to allowing you to customize the health check path in
+  // an Apollo Server web framework integration package. You're already using
+  // Express --- just define a health check yourself! This option only exists to
```

Comment:
Agree with the message but I think the way it's written might trivialize this a bit. Can we link out to some example or something?